### ISSUE TITLE
Stability Matrix Requesting Permissions for unrelated applications

### ISSUE THREAD

**Issue by thetrebor** [LykosAI/StabilityMatrix#544]
![Screenshot 2024-03-25 at 8 32 40 AM](https://github.com/LykosAI/StabilityMatrix/assets/305910/e1d5c130-eed5-46d9-ad4e-fcf795439a6c)

On OSX Stability Matrix asked for permission to access other applications, including a mobile game such as Epic Seven.
It is not desirable for the application to be working so far outside its perimeter.  Probably a few checkboxes are clicked wrong in the OSX build system.

**Comment by ionite34** [comment]
I believe this is due to our customizable Data folder location? And the nature of how python has access to all system files. I have no idea why it would show as needing to specifically access data from a game. 

https://github.com/LykosAI/StabilityMatrix/blob/main/Build/AppEntitlements.entitlements

Our only entitlement for CI app builds is to allow JIT code execution as required for the .NET runtime to function.

**Comment by thetrebor** [comment]
Odd... so the behavior is see is 
On OSX Sonoma 14.2 on Mac Studio M2 Ultra
I have stability matrix installed, with a data folder pointed to 
~/Library/Application Support/StabilityMatrix
Checkpoints folder pointed to
~/Projects/SDModels

Then I start EpicSeven on the computer and I get the popup.  At least a few times.  This morning I did not.
I also noticed it for a few others apps.

I don't see any convergence in access between them and it's def Stability Matrix that's triggering the entitlement pop.
I looked in Application Support and didn't see any data cache for Epic Seven by app or publisher.  

I'd keep an eye open for this and see if there's a common factor.  Maybe mobile apps running on apple silicon trigger it.
